Resources/Prototypes/_Impstation/Species/Thavens/Moods/yes_and.yml

```diff
@@ -168,12 +168,6 @@
   moodName: thaven-mood-prometheus-name
   moodDesc: thaven-mood-prometheus-desc
 
-# You must openly carry a lethal weapon at all times, even if you cannot use it - it is a great dishonor to be unarmed.
-- type: thavenMood
-  id: Klingon
-  moodName: thaven-mood-klingon-name
-  moodDesc: thaven-mood-klingon-desc
-
```

Comment:
comment this out with a DeltaV comment explaining why instead of straight up deleting it